### PR TITLE
Fixed Styling for Profile badge

### PR DESCRIPTION
**Pull request by esonnino**
Idea for #166983
image

@sandy081, if you feel strongly about having the badge and the profile indicator, we can experiment with the profile indicator shift to the left when there's a badge active:

image

IMO the badge can stay always on the right side and temporarily vanish while there's a badge on the settings gear :)

### CODE REVIEW

**Review by TylerLeonhardt — DISMISSED** [review]

**Comment by isidorn** [comment]
Thanks for polishing this styling. I like it.
Let's wait for Monday when @sandy081 gets back from vacation.

**Comment by sandy081** [comment]
I like the changes. We can go ahead with hiding the profile badge when there is activity. We can listen to feedback from users and experiment accordingly.

Pushed a change to hide the profile badge when there is activity.

**Review by sandy081 — APPROVED** [review]

**Comment by sandy081** [comment]
@esonnino Feel free to merge the change after getting fresh approval

**Review by isidorn — APPROVED** [review]

**Comment by esonnino** [comment]
Thanks @sandy081! You rock ;)

**Pull request merged**
